docs/src/pages/customization/palette/palette.md

```diff
@@ -4,7 +4,7 @@
 
 ## Palette colors
 
-A color intention is a mapping of a palette color to a given intention within your application.
+A color is a mapping of a palette to a given meaning within your application.
```

Comment:
I don't think that it's clearer, I don't understand what the first nor the second sentence mean. Maybe

```suggestion
A "palette color" is a mapping of colors to a given intention within your application.
```